test/system/server_tests.rb

```diff
@@ -320,7 +320,11 @@ class WithEnvIpAndPortTests < SystemTests
     setup do
       # get our current ip address, need something different than 0.0.0.0 and
       # 127.0.0.1 to bind to
-      ENV['SANFORD_IP']   = IPSocket.getaddress(Socket.gethostname)
+      ENV['SANFORD_IP'] = Socket.getaddrinfo(
+        Socket.gethostname,
+        80,
+        Socket::AF_INET
+      ).first[3]
```

Comment:
@jcredding just curious - what prompted this change?  I'm fine with it - just curious.